system/jlib/jplane.cpp

```diff
@@ -336,10 +338,25 @@ class CStoragePlane final : public CInterfaceOf<IStoragePlane>
 
     const char * queryCategory() const { return category.c_str(); }
 
+    virtual bool isCompressed(bool defaultValue) const override
+    {
+        if (compression)
+            return true;
+
+        bool compressed = defaults->getPropBool("@compressLogicalFiles", defaultValue);
+        return config->getPropBool("@compressLogicalFiles", compressed);
```

Comment:
should this (@compressLogicalFiles) be checked in ctor instead, as @compressed/other props. ?

Reply:
The problem is knowing whether it was set or not.  I will revisit though.